docs/examples/google-core-iot/README.md

```diff
@@ -25,30 +25,51 @@ $ gcloud iot registries create REGISTRY_ID \
     --state-pubsub-topic=wott-pubsub
 ```
 
-That's it. We now have created a WoTT enabled Google CoreIoT registry. Now we need to enroll our first device.
+That's it. We have now created a WoTT enabled Google CoreIoT registry. Now we need to enroll our first device.
 
 
 ## Enrolling devices
 
-The first thing we need to do is to download our the certificate of the device. To do that we issue an API call to WoTT's API:
+The first thing we need to do is to download the certificate of the device. To do that we nned to issue an API call to WoTT's API.
+To do this, you will need the Device ID of the WoTT agent-enabled device. The relevant information of your device can be found on the WoTT Dash. 
+
+If you do not have the dash set up, you can manually retrieve this information via command line using: `$ sudo wott-agent whoami` and substitute that value into `mydevice` as follows:
 
 
 ```
 $ export DEVICE_ID=mydevice.d.wott.local
-$ curl -s "https://api.wott.io/v0.2/device-cert/$DEVICE_ID" | jq -r '.certificate' > device.crt
+$ curl -s "https://api.wott.io/v0.2/device-cert/$DEVICE_ID"  > device.crt
 ```
 
-With the certificate downloaded, we can now enroll the device:
+If your WoTT ID starts with a number, do the following commands instead:
+
+
+```
+$ export DEVICE_ID=mydevice.d.wott.local
+$ export GOOGLE_DEVICE_ID=$(echo $DEVICE_ID | sed 's/^[0-9]/a-/g')
+$ curl -s "https://api.wott.io/v0.2/device-cert/$DEVICE_ID"  > device.crt
 
 ```
-$ gcloud iot devices create "$DEVICE_ID" \
+
+This achieves the same as before but gives you a valid Google Device ID that you can use to communicate with Google's services.
+
+**Note:** Google's Device ID [must start with a letter ([a-zA-Z]))](https://cloud.google.com/iot/docs/requirements#permitted_characters_and_size_requirements).
```

Comment:
@fshmcallister please reference that we are doing this above in the `GOOGLE_DEVICE_ID` reference.

Reply:
done